### ISSUE TITLE
How can I just use the net_G_A.pth when I test my model?

### ISSUE THREAD

**Issue by QHLang** [junyanz/pytorch-CycleGAN-and-pix2pix#1139]
Hello. 
When I test my trained-model, I will expend lots of time to get the fake images, because my images of test set are so many.  And I just want to translate testA to testB, so I just want to use net_G_A.pth without using net_G_B.pth, for saving time.
So could you please tell me how I can just use the net_G_A.pth when I test my model?  Thanks a lot!

**Comment by junyanz** [comment]
You can use the `--model test`.  See the [README](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix#apply-a-pre-trained-model-cyclegan) for more details. You need to manually rename the model file.

**Comment by QHLang** [comment]
OK, I see. Thanks a lot.
